src/Elastic.Apm.EntityFrameworkCore/ElasticEntityFrameworkCoreListener.cs

```diff
@@ -12,8 +13,8 @@ public class ElasticEntityFrameworkCoreListener
 		/// <summary>
 		/// Start listening for EF Core diagnosticsource events
 		/// </summary>
-		public void Start() => DiagnosticListener
+		public void Start(AbstractLogger logger) => DiagnosticListener
```

Comment:
Similarly to `ElasticCoreListeners.cs` I think we should make this optional and default to `Agent.Config.Logger` (or `Agent.Config` if we change it...).